jsapp/js/assetUtils.ts

```diff
@@ -269,40 +270,40 @@ export function isAssetPublic(permissions?: Permission[]) {
  * For getting the icon class name for given asset type. Returned string always
  * contains two class names: base `k-icon` and respective CSS class name.
```

Comment:
Please update this comment to match new behavior

Reply:
Updated the comment :)